src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final Color[] colors = Color.values();
```

Comment:
The `colors` array should not be static if it's going to be used by an instance method `getRandomColor`. If you need only one instance of `ColorSupplier`, consider making `colors` a non-static field.